src/rotations3d.jl

```diff
@@ -22,7 +23,9 @@ end
 `Rot3DCoeffs: ` storing recursively precomputed coefficients for a
 rotation-invariant basis.
 """
-struct Rot3DCoeffs{T}
+abstract type R3DC{T} end
+
```

Comment:
Is it actually useful to introduce this abstract type? Wouldn't it be better to have a single type that constructs all possible coupling coefficients based on symmetries of the basis and of the property? Maybe ok to leave it as is to merge and then the three of us spend an afternoon unifying?

Reply:
ok - I see this is because you didn't want to delete the original implementation. We need to test this more and before we delete mine I think.